src/sections/Meshery/Meshery-integrations/Individual-Integration/IntegrationPage.style.js

```diff
@@ -0,0 +1,144 @@
+import styled from "styled-components";
```

Comment:
No need to export CSS for each section as individual divs.
Just export one `div` which should be the parent wrapper and use class for the remaining sections to apply CSS